homeassistant/components/obihai/sensor.py

```diff
@@ -50,12 +50,17 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     line_services = pyobihai.get_line_state(host, username, password)
 
+    call_direction = pyobihai.get_call_direction(host, username, password)
```

Comment:
We should check that the credentials are correct. If not we should log error and return. I missed this in the last PR.

Reply:
Ok we are now checking credentials and only continuing if the credentials are valid.  If not valid we print an error and return.